js/source/scripts.js

```diff
@@ -10,27 +10,128 @@
 // wrapping it with an "anonymous closure". See:
 // - https://drupal.org/node/1446420
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
-(function (Drupal, $, window) {
```

Comment:
I've now included the missing dependency to get this to work. It makes sense to return it back to the three parameter wrapper.